### PR TITLE
docs: normalised versions and explained releases

### PR DESCRIPTION
**Pull request by thesephirot**
aligned docs to latest/stable releases and added release explanations

### CODE REVIEW

File: docs/docs/guides/deploy/overview.mdx [hunk 1]
```diff
@@ -17,3 +17,12 @@ By default, it runs a highly available ZITADEL instance along with a secure and
 - ZITADEL does not need many resources, 1 CPU and 512MB memory are more than enough. (With more CPU, the password hashing might be faster)
 - A cockroachDB or [🚧 Postgresql coming soon](https://github.com/zitadel/zitadel/pull/3998) as only needed storage
 - If you want to front ZITADEL with a reverse proxy, web application firewall or content delivery network, make sure to support [HTTP/2](../manage/self-hosted/http2)
+
+
+## Releases
+
+The easiest way to use ZITADEL is to run one of our container releases
```
Comment (fforootd) [inline docs/docs/guides/deploy/overview.mdx:24]: Why should a container be better then the rest?

File: docs/docs/guides/deploy/macos.mdx [hunk 2]
```diff
@@ -27,9 +27,14 @@ brew install zitadel/tap/zitadel
...
+Download the ZITADEL release according to your architecture from [Github](https://github.com/zitadel/zitadel/releases/latest)
+
+## Unpack the archive
+move to your download location and unpack the archive
+
 ```bash
-export MY_ARCHITECTURE="arm64 or amd64 depeding on your mac"
-curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0 | grep "browser_download_url.*zitadel_Darwin_${MY_ARCHITECTURE}.tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+#unpack and set executable
+tar xfv zitadel_* && chmod +x zitadel
```
Comment (fforootd) [inline docs/docs/guides/deploy/macos.mdx:37]: Please create a command that downloads and creates a usable binary

File: docs/docs/guides/deploy/linux.mdx [hunk 3]
```diff
@@ -19,8 +19,14 @@ cockroach start-single-node --insecure --background --http-addr :9090 --listen-a
...
 
+Download the ZITADEL release according to your architecture from [Github](https://github.com/zitadel/zitadel/releases/latest) 
+
+## Unpack the archive
+move to your download location and unpack the archive
+
 ```bash
-curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0 | grep "browser_download_url.*zitadel_Linux_$(uname -m).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+#unpack and set executable
+tar xfv zitadel_* && chmod +x zitadel
```
Comment (fforootd) [inline docs/docs/guides/deploy/linux.mdx:29]: Please create a command that downloads and creates a usable binary

File: docs/docs/guides/deploy/overview.mdx (hunk 1 above)
Comment (thesephirot) [reply docs/docs/guides/deploy/overview.mdx:24]: I didn't say "best" but easiest. To me containers are the fastest/most convenient way to get ZITADEL  up and running

File: docs/docs/guides/deploy/macos.mdx (hunk 2 above)
Comment (thesephirot) [reply docs/docs/guides/deploy/macos.mdx]: done

File: docs/docs/guides/deploy/linux.mdx (hunk 3 above)
Comment (thesephirot) [reply docs/docs/guides/deploy/linux.mdx]: done

File: docs/docs/guides/deploy/linux.mdx [hunk 4]
```diff
@@ -19,8 +19,15 @@ cockroach start-single-node --insecure --background --http-addr :9090 --listen-a
...
+Download the ZITADEL release according to your architecture from [Github](https://github.com/zitadel/zitadel/releases/latest) 
+
+## Unpack the archive
+move to your download location and unpack the archive
+
 ```bash
-curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0 | grep "browser_download_url.*zitadel_Linux_$(uname -m).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+#unpack and copy to /usr/local/bin
+LATEST=$(curl -i https://github.com/zitadel/zitadel/releases/latest | grep location: | cut -d '/' -f 8 | tr -d '\r'); wget -qO-  https://github.com/zitadel/zitadel/releases/download/$LATEST/zitadel_Linux_$(uname -m).tar.gz  | tar -xJ zitadel && sudo mv zitadel /usr/local/bin
+
```
Comment (fforootd) [inline docs/docs/guides/deploy/linux.mdx:30]: This command is not working on an ubuntu 😁

Comment (fforootd) [reply docs/docs/guides/deploy/linux.mdx:30]: image

Comment (thesephirot) [reply docs/docs/guides/deploy/linux.mdx:30]: thanks, I will test and fix this in an ubuntu container

Comment (thesephirot) [reply docs/docs/guides/deploy/linux.mdx]: fixed

**Review by fforootd — COMMENTED** [review]
I could not test the mac command but the other worked

**Review by fforootd — APPROVED** [review]

**Comment by fforootd** [comment]
@thesephirot go ahead and merge this if you feel ready

**Review by fforootd — APPROVED** [review]

**Comment by thesephirot** [comment]
> I could not test the mac command but the other worked

I did tests on Mac already, that will work

**Pull request merged**